Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -375,6 +375,24 @@ public WilliamsPercentR WILR(string symbol, int period, Resolution? resolution =
             RegisterIndicator(symbol, williamspercentr, resolution, selector);
             return williamspercentr;
         }
+
+        /// <summary>
+        /// Creates a new On Balance Volume indicator. This will compute the 
```

Comment:
Looks like we never got around to finishing up this doc.  This will compute the... (I'm not sure what to put, maybe steal some verbiage from [wikipedia's OBV article](http://en.wikipedia.org/wiki/On-balance_volume)